HELSSA_AGENTS/TEMPLATES/app_template/app_code/cores/api_ingress.py

```
@@ -0,0 +1,282 @@
+        try:
+            # Basic validation
+            if not isinstance(request_data, dict):
+                raise ValueError("درخواست باید یک JSON object باشد")
+            
+            # Sanitize input data
+            validated_data = self._sanitize_input(request_data)
+            
+            # Apply business validation rules
+            validated_data = self._apply_business_validation(validated_data)
+            
+            self.logger.info(f"Request validation successful for {APP_NAME}")
```

Ruff (0.12.2)

Undefined name `APP_NAME`

(F821)